src/Jeffijoe.MessageFormat/Formatting/Formatters/PluralFormatter.cs

```diff
@@ -220,86 +220,94 @@ internal string Pluralize(string locale, ParsedArguments arguments, PluralContex
         /// <returns>
         ///     The <see cref="string" />.
         /// </returns>
-        internal string ReplaceNumberLiterals(StringBuilder pluralized, double n)
+        internal string ReplaceNumberLiterals(string pluralized, double n)
```

Comment:
I wonder, is there a way to use `Span` for perf here?

Reply:
Here, we get the value from the cached strings, so I think we can't

Reply:
I think we can try using Span APIs via changing for index StringBuilder loops to GetChunks in NET5.0, but that's a piece of work in itself )

Reply:
Ah, so you think `GetChunks` is faster than an index accessor?

Reply:
Yeah, index accessor basically does a lot of theduplicating work when iterating through the whole StringBuilder (finds the chunk where the idx is each time)
See here https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/Text/StringBuilder.cs#L497

Reply:
Ah nice find! If we can run the tests against .NET 5 as well as the one that does not support it, I would not be opposed to having 2 different implementations of the most critical paths. Alternatively, we can just bump support to the lowest target that supports `Span` (even in its less optimal form which should still be more performant)

Reply:
The GetChunks API is available starting from netcoreapp3.1. 
I prefer multitargeting for now, because netstandard2.1 does not have the APIs required, and the only next version after netstandard2.0 (that supports both Mono and CoreCLR) is net5.0